src/SmartComponents/Recs/List.js

```diff
@@ -20,13 +21,26 @@ let AdvisorRedHatDisabledRuleAlert = sessionStorage.getItem('AdvisorRedHatDisabl
 const List = () => {
     const intl = useIntl();
     const [redhatDisabledRuleAlert, setRedHatDisabledRuleAlert] = useState(AdvisorRedHatDisabledRuleAlert);
+    const [redHatDisabledRuleCount, setRedHatDisabledRuleCount] = useState(0);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const response = (await API.get(`${RULES_FETCH_URL}?rule_status=rhdisabled`)).data;
+                setRedHatDisabledRuleCount(response.meta.count);
+            } catch (error) {
+                setRedHatDisabledRuleCount(0);
+                throw `${error}`;
```

Comment:
m thinking lets not throw an error over this fail let it be silent n just not show the message? 🤔

Reply:
Done and done!